css/dashboard.css

```diff
@@ -72,20 +56,37 @@ ul.request {
   display: -webkit-flex;
   display: flex;
   -webkit-align-items: stretch;
-  align-items: stretch;
+  align-items: center;
   margin-top:8px;
-  width:100%;
-
+  width: 100%;
+  margin-left: 14px;
+  padding-right: 68px;
 }
 .search input{
   height: 36px;
-  padding:8px;
-  width:100%;
 }
 .search input:focus {
   border-color: rgb(119,119,119);
   box-shadow: none;
 }
+.menu-drawer {
+  display: block;
+  position: fixed;
+  padding: 12px;
+  color: white;
+  background-color: #C8254D;
+  border-radius: 10%;
+  top: 30%;
+  left: 2%;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  cursor: pointer;
+  z-index: 2;
+  transition: top 0.3s;
```

Comment:
Nope, only stick to transiotion of `transform` and `opacity` as they don't trigger `Paint`.